vtex/billing-app
# Create contract status check query

#### What is the purpose of this pull request?
Create a status check query for an app contract

#### What problem is this solving?
This adds a consult feature to the billing app that can be useful for apps that are charged

#### How should this be manually tested?
1. Clone/pull this repo/branch and link this app (`vtex.billing`) in your workspace
2. In a test app, add this policy:
```
{
  "name": "vtex.billing:contract-status"
}
```
And this dependency:
```
{
  "vtex.billing": "0.x"
}
```
3. From a test app, try to call this endpoint:
`http://billing.vtex.{{region}}.vtex.io/{{account}}/{{workspace}}/_v/isContractActive`
This should return a boolean value that shows whether your app's contract is active or not

#### Types of changes
